    Change the source-highlighter to pygments as we don't have the source-highlighter package available yet:

--- asciidoc-9.0.4/asciidoc.conf.orig	2020-10-30 16:11:52.000000000 +0000
+++ asciidoc-9.0.4/asciidoc.conf	2020-11-07 08:48:43.372582348 +0000
@@ -22,7 +22,7 @@
 # Uncomment to use xhtml11 quirks mode CSS.
 #quirks=
 # HTML source code highlighter (source-highlight, pygments or highlight).
-source-highlighter=source-highlight
+source-highlighter=pygments
 # Uncomment to use deprecated quote attributes.
 #deprecated-quotes=
 empty=
